server/channels/api4/preference_test.go

```diff
@@ -255,6 +279,21 @@ func TestUpdatePreferences(t *testing.T) {
 	resp, err = client.UpdatePreferences(context.Background(), user1.Id, preferences1)
 	require.Error(t, err)
 	CheckUnauthorizedStatus(t, resp)
+
+	//UpdatePreferences API from System Admin and Local Client
```

Comment:
Could you please add a space before each comment as that is the norm in this codebase?

Reply:
Done